web/src/pages/Proto/locales/en-US.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export default {
+  'page.proto.list': 'Proto List',
+  'page.proto.list.description':
+    "Protocol buffers are Google's language-neutral, platform-neutral, extensible mechanism for serializing structured data.You define how you want your data to be structured once, then you can use special generated source code to easily write and read your structured data to and from a variety of data streams and using a variety of languages.",
```

Comment:
The relationship between the `proto` and the `grpc-transcode` plugin needs to be explained.

Reply:
you could refer to https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/grpc-transcode.md

Reply:
Hi @oil-oil , we must explain that the proto resources here are used in the `grpc-transcode` plugin and that the proto resources created here can be accessed by selecting the corresponding IDs when the grpc-transcode plugin is opened. There is no need to explain what Protocol buffers are.

Reply:
@oil-oil  I see your update. LGTM, but should be better to keep the previous introduction.